test/unit/helpers/check_ip_in_list.js

```diff
@@ -41,27 +41,15 @@ describe('checkIpInList', () => {
 			done();
 		});
 
-		describe('when returnListIsEmpty is not a boolean', () => {
-			before(done => {
-				validReturnListIsEmpty = null;
-				done();
-			});
-
-			it('should set returnListIsEmpty to true', done => {
-				expect(checkIpInListResult).to.eq(true);
-				done();
-			});
-		});
-
 		describe('when validList is not an array', () => {
 			before(done => {
 				validReturnListIsEmpty = true;
 				validList = null;
 				done();
 			});
 
-			it('should return validReturnListIsEmpty', done => {
-				expect(checkIpInListResult).to.eq(validReturnListIsEmpty);
+			it('should return false', done => {
```

Comment:
We should update the beforeEach here: https://github.com/LiskHQ/lisk/pull/2344/files#diff-3ea1e3ed6b5ed11246f4d2b735f21bd8R39 and remove `validReturnListIsEmpty` usages.